dask/dataframe/accessor.py

```diff
@@ -142,7 +142,11 @@ def cat(self, others=None, sep=None, na_rep=None):
         from .core import Index, Series
 
         if others is None:
-            raise NotImplementedError("x.str.cat() with `others == None`")
+
+            def str_cat_none(x):
+                return pd.Series(x).str.cat(sep=sep, na_rep=na_rep)
```

Comment:
I am wondering if this will work well with cudf (@quasiben can you comment?)

It seems like this would achieve the same outcome right?

```suggestion
                if isinstance(x, (Series, Index)):
                    x = x.compute()
                return x.str.cat(sep=sep, na_rep=na_rep)

Reply:
@jsignell  Thanks for the suggestion and for reviewing the PR